Resources/Prototypes/Nuclear14/Entities/Objects/Weapons/Ammunition/AmmunitionCasings.yml

```diff
@@ -0,0 +1,101 @@
+- type: entity
+  id: genericCasing
+  parent: BaseItem
+  abstract: true
+  components:
+  - type: Sprite
+    netsync: false
+  - type: Tag
+    tags:
+    - Cartridge
+    - GCasing
+  - type: Item
+    size: 1
+  - type: Recyclable
+  - type: SpaceGarbage
+  proto: genericCasing
+
+- type: entity
+  id: 308roundcasing
+  parent: BaseItem
+  abstract: true
+  components:
+  - type: Sprite
+    netsync: false
+  - type: Tag
+    tags:
+    - Cartridge
+    - Casing
+  - type: Item
+    size: 1
+  - type: Recyclable
+  - type: SpaceGarbage
+  proto: 308roundCasing
+  
+ - type: entity
+  id: 44roundcasing
+  parent: BaseItem
+  abstract: true
+  components:
+  - type: Sprite
+    netsync: false
+  - type: Tag
+    tags:
+    - Cartridge
+    - Casing
+  - type: Item
+    size: 1
+  - type: Recyclable
+  - type: SpaceGarbage
+  proto: 44roundCasing
+
+ - type: entity
+  id: 5.56mmroundcasing
+  parent: BaseItem
+  abstract: true
+  components:
+  - type: Sprite
+    netsync: false
+  - type: Tag
+    tags:
+    - Cartridge
+    - Casing
+  - type: Item
+    size: 1
+  - type: Recyclable
+  - type: SpaceGarbage
+  proto: 5.56mmroundCasing
+
+- type: entity
+  id: 10mmroundcasing
+  parent: BaseItem
+  abstract: true
+  components:
+  - type: Sprite
+    netsync: false
+  - type: Tag
+    tags:
+    - Cartridge
+    - Casing
+  - type: Item
+    size: 1
+  - type: Recyclable
+  - type: SpaceGarbage
+  - proto: 10mmroundCasing
```

Comment:
again, this can't be in component registries

Reply:
Ah, I thought I updated the Casing entities as well - My mistake.